app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ContactsController < ApplicationController
+  before_action :contact_validation, only: [:create]
+
+  def create
+    @contact = contact_params
+    ContactMailer.user_email(@contact).deliver_now
+    ContactMailer.admin_email(@contact).deliver_now
```

Comment:
先に管理者に送信した方が無難だと思います。
ユーザーにだけ送信して管理者送信に失敗するのはかなりまずいパターンですので。

```suggestion
    ContactMailer.admin_email(@contact).deliver_now
    ContactMailer.user_email(@contact).deliver_now
```